tracing/src/main/java/com/palantir/tracing/Traces.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.tracing;
+
+import com.google.common.base.Strings;
+
+public final class Traces {
+
+    public static final String TRACE_HEADER = "trace-id";
```

Comment:
I think we want something like `X-B3-TraceId` which is used by both [zipkin](https://twitter.github.io/zipkin/Instrumenting.html) and [Spring Cloud Sleuth](https://github.com/spring-cloud/spring-cloud-sleuth/blob/master/spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java#L52)

See also:
* https://spring.io/blog/2016/02/15/distributed-tracing-with-spring-cloud-sleuth-and-spring-cloud-zipkin
* http://cloud.spring.io/spring-cloud-sleuth/spring-cloud-sleuth.html#_feign